assets/docs/configuration/targets/http-full-example.hcl

```diff
@@ -38,6 +38,8 @@ target {
     # Whether to skip verifying ssl certificates chain (default: false)
     # If tls_cert and tls_key are not provided, this setting is not applied.
     skip_verify_tls            = true
+
+    # Whether to enable setting headers dynamically
+    dynamic_headers            = true
```

Comment:
Not strictly needed, we could indeed imply this as you said.
I was thinking it may be nice to have a way to disable this behaviour. Thinking cases where a tracking implementation mistake may take more time to fix than just disabling from here.
It feels more safe to me being extra-explicit eitherway, but i may be missing an argument for the opposite.
What do you think?

Reply:
I hadn't thought of that - it's good defensive thinking. I agree, let's keep the config.